server/src/store.js

```diff
@@ -8,34 +8,40 @@ const DEFAULT_PAGE_SIZE = 25;
 mongoose.Promise = global.Promise;
 mongoose.connect(process.env.STORE_URI);
 
+export function getMongooseConnection() {
+  return mongoose.connection;
+}
+
 export function queryBit({ bitId }) {
   return Bit.findOne({ postId: bitId }).exec();
 }
 
 export function queryBits({
-    sort=SORT_PARAM_DATE,
-    offset=0,
-    limit=DEFAULT_PAGE_SIZE,
-    mainChars,
-    vsChars,
-    stages,
-    standaloneTags,
-  } = {}) {
+  sort = SORT_PARAM_DATE,

```

Comment:
whaaaatt even is this syntax

Reply:
It's how you do named parameters: https://javascript.info/destructuring-assignment#smart-function-parameters